4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-editor-language-can-switch-on-demand.md

---
type: entry
section: general
subsection: editor
rank: 3
importance: 4
anchor: editor-language-can-switch-on-demand
title: Editor language can switch on-demand
text: |
  Changing the editor language no longer requires an editor restart.

>   That's now the occasion to test the editor in all the languages it supports!

Reviewer: It's also useful for testing translation of editor plugins. (though this feature is not much documented)

Author: Done!

>   That's now the occasion to test the editor in all the languages it supports!

Reviewer 2: ```suggestion
  That’s now the occasion to test the editor in all the languages it supports!
```

>   That's now the occasion to test the editor in all the languages it supports!

Reviewer 3: ```suggestion
  Now you can test the editor in all the languages it supports!
```

---
type: entry
section: general
subsection: editor
rank: 3
importance: 4
anchor: editor-language-can-switch-on-demand
title: Editor language can switch on-demand
text: |
  Changing the editor language no longer requires an editor restart.

>   For editor plugin developers, this feature can be quite handy in order to test your translations.

Reviewer 4: ```suggestion
  This feature can be quite handy for editor plugin developers testing their translations.
```

>   For everyone else, that's now the occasion to test the editor in all the languages it supports!

Reviewer 2: Not sure the term "that's now the occasion to" is an established English phrase, I can't find it anywhere. The closest would be "have an occasion" but that means something that's necessary to do, so I'd suggest:
```suggestion
  For everyone else, now’s the time to seize the occasion to test the editor in all the languages it supports!
```

Author: I used [the suggestion](https://github.com/godotengine/godot-website/pull/1138#discussion_r2267406773) of @JohnVeness
